client/src/components/Map/index.tsx

```diff
@@ -12,12 +11,23 @@ import envData from '../../../../config/env.json';
 
 const { curriculumLocale } = envData;
 
-const propTypes = {
-  currentSuperBlock: PropTypes.string,
-  forLanding: PropTypes.bool
-};
+interface MapProps {
+  currentSuperBlock?: string;
+  forLanding?: boolean;
+}
+
+interface Node {
+  superBlock: SuperBlock;
+  dashedName: string;
+}
+
+interface MapData {
+  allChallengeNode: {
```

Comment:
Would you mind refactoring this to use the globally-defined prop-types: https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/redux/prop-types.ts#L172